src/keyboardAction.js

```diff
@@ -227,17 +280,23 @@ export function dndzone(node, options) {
                 .filter(dz => dz === focusedDz || !dzToConfig.get(dz).dropFromOthersDisabled),
             dz => dzToConfig.get(dz).dropTargetStyle,
         );
-        dispatchConsiderEvent(node, dzToConfig.get(node).items, {trigger: TRIGGERS.DRAG_STARTED, id: focusedItemId, source: SOURCES.KEYBOARD});
+        config.itemsAtDragStart =  dzToConfig.get(node).items
```

Comment:
i think there is no need to get it as a config.  you are persisting it on drag start, which is the right thing to do. instead of storing it in the config you can store it as a global like everything else that's related to a drag operation (since there is only one of these at any given time)